docs/content/config/security/rspamd.md

```diff
@@ -2,37 +2,87 @@
 title: 'Security | Rspamd'
 ---
 
-!!! warning "Implementation of Rspamd into DMS is WIP!"
+!!! warning "The Current State of Rspamd's Integration Into DMS"
+
+    Recent pull requests have stabilized Rspamd's integration to a point where we encourage users to test the feature. We are confident that there are no major bugs in our code, that make using Rspamd infeasible, anymore. Please note that there may still be (breaking) changes ahead as integration is still work in progress!
```

Comment:
```suggestion
!!! warning "The current state of Rspamd integration into DMS"

    Recent pull requests have stabilized integration of Rspamd to a point that we encourage users to test the feature. We are confident that there are no major bugs in our integration that make using Rspamd infeasible. Please note that there may still be (breaking) changes ahead as integration is still work in progress!
```

Reply:
Done in 62e528d380ac92f6cc2e361c4685c01780d38088